lib/services/local/users.go

```diff
@@ -1517,25 +1517,27 @@ func KeyAttestationDataFingerprintV11(pub crypto.PublicKey) (string, error) {
 }
 
 const (
-	webPrefix                 = "web"
-	usersPrefix               = "users"
-	sessionsPrefix            = "sessions"
-	attemptsPrefix            = "attempts"
-	pwdPrefix                 = "pwd"
-	connectorsPrefix          = "connectors"
-	oidcPrefix                = "oidc"
-	samlPrefix                = "saml"
-	githubPrefix              = "github"
-	requestsPrefix            = "requests"
-	requestsTracePrefix       = "requestsTrace"
-	usedTOTPPrefix            = "used_totp"
-	usedTOTPTTL               = 30 * time.Second
-	mfaDevicePrefix           = "mfa"
-	webauthnPrefix            = "webauthn"
-	webauthnGlobalSessionData = "sessionData"
-	webauthnLocalAuthPrefix   = "webauthnlocalauth"
-	webauthnSessionData       = "webauthnsessiondata"
-	recoveryCodesPrefix       = "recoverycodes"
-	recoveryAttemptsPrefix    = "recoveryattempts"
-	attestationsPrefix        = "key_attestations"
+	webPrefix                   = "web"
+	usersPrefix                 = "users"
+	sessionsPrefix              = "sessions"
+	attemptsPrefix              = "attempts"
+	pwdPrefix                   = "pwd"
+	connectorsPrefix            = "connectors"
+	oidcPrefix                  = "oidc"
+	samlPrefix                  = "saml"
+	githubPrefix                = "github"
+	requestsPrefix              = "requests"
+	requestsTracePrefix         = "requestsTrace"
+	usedTOTPPrefix              = "used_totp"
+	usedTOTPTTL                 = 30 * time.Second
+	mfaDevicePrefix             = "mfa"
+	webauthnPrefix              = "webauthn"
+	webauthnGlobalSessionData   = "sessionData"
+	webauthnLocalAuthPrefix     = "webauthnlocalauth"
+	webauthnSessionData         = "webauthnsessiondata"
+	recoveryCodesPrefix         = "recoverycodes"
+	recoveryAttemptsPrefix      = "recoveryattempts"
+	attestationsPrefix          = "key_attestations"
+	assistantMessagePrefix      = "assistant_message"
+	assistantConversationPrefix = "assistant_conversation"
```

Comment:
Nit: Make these plural for consistency with other keys (users, sessions, etc.).
```suggestion
	assistantMessagesPrefix      = "assistant_messages"
	assistantConversationsPrefix = "assistant_conversations"
```

Reply:
Changed.